definitions/scenarios/packages.rb

```diff
@@ -1,5 +1,14 @@
 module ForemanMaintain::Scenarios
   module Packages
+    def self.skip_installer_run?(packages_list)
+      return false unless packages_list.any? { |p| p.include?('foreman_maintain') }
+      return true if packages_list.length == 1
+
+      fm_pkg = ForemanMaintain.main_package_name
+      puts "ERROR: Perform this action for '#{fm_pkg}' before any other packages."
```

Comment:
Should we change error message something like:
"ERROR: Install or Update '#{fm_pkg}' individually."

Reply:
Surely! Big +1

Reply:
Done!